woocommerce/woocommerce
# Fix `Product Meta` translations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR registers the Product Meta block server side to fix translations (following the same idea that was done in this PR https://github.com/woocommerce/woocommerce-blocks/pull/6737 to fix cart translations).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout trunk.
2. Switch to the Italian or Spanish language (or some language that has 100% of string translated).
3. Visit `wp-admin/site-editor.php?postId=woocommerce%2Fwoocommerce%2F%2Fsingle-product&postType=wp_template&canvas=edit`.
4. Check the `Product Meta`.
5. Ensure that the block isn't translated.
6. Checkout this branch.
7. Repeat 2-4.
8. Ensure that the block is translated.
9. Create a new page or post and insert the `Single Product` block.
10. Check the `Product Meta` block is also translated here.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Fix "Product Meta" translations - Register the block server side.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
